common/beerocks/bcl/include/bcl/beerocks_defines.h

```diff
@@ -443,6 +447,30 @@ enum eBssType {
     BSS_TYPE_INVALID
 };
 
+enum class eChannelScanErrCode : uint8_t {
```

Comment:
Since its an enum class, there is no need to have the `CHANNEL_SCAN_` prefix

Reply:
#665